Presentation.Web/app/components/it-system/it-system-catalog.controller.ts

```diff
@@ -901,11 +905,11 @@
             return this.$(name) as any;
         }
 
-        public beginItSystemMigration = (municipalityId: number, municipalityName: string, usageId: number) => {
+        public beginItSystemMigration = (municipalityUuid: string, municipalityName: string, usageUuid: string) => {
```

Comment:
ok so we may have to do some adjustments in the response..

Notice that the uuids are different in this screenshot: 
![image](https://user-images.githubusercontent.com/13116889/224925328-de20f103-4f36-4155-99ab-bdb02c2851b3.png)

We need to
- Change the From/To references in the relation mappings to point to the master system, so that the uuids are correct and can actually be found in the UI by the users getting the notification. Right now the "to/from" map the system usages and their uuids which is ok in the application layer but in the DTO layer we must now use the master system in stead and also change the property names accodingly "FromSystem and ToSystem" in stead of usage. Change should be simple though.. Different mapping, naming and so on